db/migrate/20170901060759_create_wind_turbine_speed_measurements.rb

```diff
@@ -0,0 +1,8 @@
+class CreateWindTurbineSpeedMeasurements < ActiveRecord::Migration[5.0]
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?